manifests/package.pp

```diff
@@ -38,19 +38,22 @@
   if $elasticsearch::ensure == 'present' {
 
     # Create directory to place the package file
-    exec { 'create_package_dir_elasticsearch':
-      cwd     => '/',
-      path    => ['/usr/bin', '/bin'],
-      command => "mkdir -p ${elasticsearch::package_dir}",
-      creates => $elasticsearch::package_dir,
-    }
+    $package_dir = $elasticsearch::package_dir
```

Comment:
In this case the package_dir var is used by the plugin define as well to have a temp location to place the plugin when it comes from a puppet:// resource for offline installations, so i think it needs to be re-implemented for OpenBSD as well.